cpp/dspace/doi.cc

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+#include <regex>
+#include <set>
+#include <unordered_map>
+#include <cstdlib>
+#include "DbConnection.h"
+#include "DbResultSet.h"
+#include "DbRow.h"
+#include "Downloader.h"
+#include "EmailSender.h"
+#include "HttpHeader.h"
+#include "JSON.h"
+#include "StlHelpers.h"
+#include "StringUtil.h"
+#include "VuFind.h"
+#include "util.h"
+
+using namespace std;
+
+const std::string DspaceServerURL("https://publikationen.uni-tuebingen.de/rest/items/");
+const std::string DOIKey("dc.relation.uri");
+const std::string PublicationTitleKey("dc.title");
+const std::vector<std::string> notification_email_addresses = { "andrii.lysohor@uni-tuebingen.de" };
+
+void SendNotificationsForDOI(const std::string email_subject, const std::string email_message) {
+    if (EmailSender::SimplerSendEmail("no-reply@ub.uni-tuebingen.de", notification_email_addresses, email_subject, email_message,
+                                      EmailSender::VERY_HIGH)
```

Comment:
We only use VERY_HIGH priority if we send error mails. Since this is just a normal mail, we can also send it with normal priority.